map_server/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(map_server)
 find_package(catkin REQUIRED
         COMPONENTS
             roscpp
+            roslib
```

Comment:
this seems to only be in the Cmake/package.xml - where is it actually used/required?